docs/site/contest_formats.md

```diff
@@ -1,22 +1,31 @@
 # Contest Formats
 
-The DMOJ ships with 4 contest formats by default: Default, Codechef IOI Ranklist (henceforth shortened to simply IOI), ECOO, and AtCoder.
+The DMOJ ships with 6 contest formats by default: Default, Codechef IOI Ranklist (henceforth shortened to simply IOI), Legacy IOI (pre-2016), ECOO, AtCoder and ICPC.
 
 ## Default Contest Format
 
 The `Default` contest format is what all contests ran on pre-April 23, 2019.
 
-The score is the sum of the maximum scores on each problem, and ties are broken by adding the time of the last submission to each problem with a non-zero maximum score.
+The score is the sum of the highest-scoring submission on each problem, and ties are broken by adding the time of the last submission to each problem with a non-zero maximum score.
 
 Note that *any* submission will increase time penalty, not just score-changing submissions.
 
 There are no additional options that can be configured for this contest format.
 
 ## IOI Contest Format
 
-The `IOI` contest format emulates the scoring used by Codechef's IOI Ranklist.
+The `IOI` contest format emulates the scoring used by Codechef's IOI Ranklist from 2016 onwards.
 
-The score is equal to the sum of the maximum scores on each problem, and by default, ties are not broken.
+The score is equal to the sum of the final score on each problem, where the final score for a problem is the maximum score for each subtask across all submissions, and by default, ties are not broken.
+For example, consider a contestant made two submissions on a task with two subtasks. The first gets 30 points on the first subtask and 10 points on the second subtask. The second gets 0 points on the first subtask and 40 points on the second subtask. The final score for this problem will be 70.
+
+The `cumtime` option can be set to `true` within the JSON configuration. This will break ties by summing the submissions times of the most recent submissions.

```

Comment:
Are you sure it's "most recent", not "the first submissions that passes each subtask"